app/views/events/index.haml

```diff
@@ -0,0 +1,48 @@
+%h2 My events
+- if current_user.events.any?
+  %ul.list-group
+    - @user_events.each do |event|
+      %li.list-group-item
+        .row
+          .col-xs-3
+            =  image_tag(event.picture, class: "img-responsive")
+          .col-xs-9
+            .media-body
+              %h3.media-heading
+                = event.title
+              .row
+                .col-xs-12.col-md-6
+                  .row.no-gutter
+                    .col-xs-3.text-right.truncate
```

Comment:
![image](https://cloud.githubusercontent.com/assets/494720/6426606/6bd3f9e4-bf64-11e4-956a-cdff02a2d073.png)
IMO static text provided by our service should be visible to the user, not truncated like that. Consider to move events details under the image on small devices :)